binding-mruby/font-binding.cpp

```diff
@@ -98,7 +102,10 @@ MRB_METHOD(FontSetName)
 	mrb_value name;
 	mrb_get_args(mrb, "S", &name);
 
-	f->setName(RSTRING_PTR(name));
+	std::vector<std::string> names;
+	names.push_back(RSTRING_PTR(name));
+
+	f->setName(names);
```

Comment:
Compare the MRI binding: We additionally save the passed argument as an IV, and return that in the getter.